python/paddle/fluid/executor.py

```diff
@@ -348,6 +348,12 @@ def _fetch_data(self, fetch_list, fetch_var_name, scope):
         ]
         return outs
 
+    def begin_pass(self):
```

Comment:
haven't looked at much detail yet. Is there any way to avoid this? It seems other framework doesn't need to distinguish different passes?